mail_activity_team/views/res_users_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<odoo>
+    <!-- Update user form !-->
+    <record id="view_users_form_activity_teams" model="ir.ui.view">
+        <field name="name">res.users.form.activity.team</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form"/>
+        <field name="arch" type="xml">
+            <data>
+                <field name="signature" position="before">
+                    <field name="activity_team_ids" widget="many2many_tags"/>
```

Comment:
The label is not displayed.
![image](https://user-images.githubusercontent.com/23449160/49229467-e0da4600-f3ed-11e8-94ad-a1fa0b2051ce.png)

Reply:
If you test in local, you see that the label is displayed. Maybe it's not displayed due to another module of the repository

Reply:
It due to `mail_digest` module